examples.md

```diff
@@ -95,4 +96,21 @@ uses: actions/cache@preview
     key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
     restore-keys: |
       ${{ runner.os }}-gem-
-```
\ No newline at end of file
+```
+
+## Python - pip
+
+Use with virtual environments.
+
+```yaml
+- uses: actions/cache@preview
+  with:
+    path: .venv
+    key: ${{ runner.os }}-pip-${{ hashFiles(format('{0}/{1}', github.workspace, 'requirements.txt')) }}
+    restore-keys: |
+      ${{ runner.os }}-pip-
+- run: |
+    python -m venv .venv
+    source .venv/bin/activate
+    python -m pip install -r requirements.txt
```

Comment:
Remove run section.

Reply:
But `.venv` is created by `python -m venv .venv` command 🤔

Reply:
This is just an example of the cache action API, not how to write a Python workflow setup.

Reply:
See examples in other languages/ecosystems.

Reply:
For an example of using virtualenv, changing to the minimum example code and title to `Python - pip, virtualenv` should enough. (imo)

Reply:
Ummm... `.venv` is not default package install path.
Default path is`/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/` for  `actions/setup-python@v1` with `python-version: 3.7`
Is `/opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/site-packages/` better cache path in examples?

Reply:
See the comment above.